pkgs/top-level/perl-packages.nix

```diff
@@ -10948,13 +10948,14 @@ let
     };
   };
 
-  LaTeXML = buildPerlPackage rec {
+  LaTeXML = lib.recursiveUpdate (buildPerlPackage rec {
```

Comment:
Maybe it is more appropriate to build `LateXML.tex` as a derived package under `pkgs/tools/misc` instead (where as I understand it, we have the bulk of (La)TeX tools in?)

Reply:
Why is this not using `passthru`? In its current state, all extra attributes will be lost on next `overrideAttrs`.

Reply:
Yes, that's not ideal. But I don't think you can pass different `passthru`'s to different outputs. Can you?

Reply:
(OTOH I might just drop the `tlType = "bin"` idea, it's too cumbersome with the current `texlive.combine`.)

Reply:
I gave up on `tlType = "bin"`. Ultimately, if LaTeXML has to depend on `dvi*`, the `makeWrapper` approach is the appropriate one. Now `makeWrapper` does not work, because `dvipng` and `dvisvgm` are not functional when not combined. But this is an issue to be addressed in `texlive.bin`, not worked around here.